### ISSUE TITLE
Pre-work Checklist: Developer: Roy Chan

### ISSUE THREAD

**Issue by roychan1** [hackforla/website#5781]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [ ] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by roychan1** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It has taken me about 4 hours in total to get to this point since the start of this issue.

**Comment by roychan1** [comment]
1. Progress: I have completed two first good issues in the past week, and just picked up my first small issue.
2. Blockers: I have not encountered any blockers so far.
3. Availability: I should be available anytime starting Saturday 11/11.
4. ETA: To complete a small issue, a medium issue, and a large issue, I am anticipating completion about one week from now.

**Comment by roychan1** [comment]
1. Progress: I have completed my small issue which is waiting on review
2. Blockers: I was occupied with other responsibilities this week so I was not able to get through issues as fast, but I will try my best to work on them this weekend
3. Availability: I am available the weekend of 11/18-11/19
4. ETA: I am anticipating another 3-5 days to complete a medium and a large issue, depending on how fast the pull requests get reviewed.

**Comment by roychan1** [comment]
1. Progress: I have completed my small issue and picked up a medium issue.
2. Blockers: I was occupied with other responsibilities this week so I was not able to get through issues as fast, but I will try my best to work on them this weekend
3. Availability: I am available the weekend of 11/25-11/26
4. ETA: I am anticipating another 3-5 days to complete a medium and a large issue, depending on how fast the pull requests get reviewed.

**Comment by roychan1** [comment]
1. Progress: I am working on my medium issue.
2. Blockers: I have a busier schedule this week, and it is taking a significant amount of time to learn and research around the topic of my issue.
3. Availability: I am available the weekend of 12/9 - 12/10
4. ETA: I am anticipating another week to finish or make significant progress through the ticket, I'll also try to attend office hour on Thursday to speed up my progress.

**Comment by roychan1** [comment]
1. Progress: I am working on my medium issue, I was trying to attend the Thursday office hour but meeting was not started.
2. Blockers: it is difficult to figure out a solution
3. Availability: I am available the weekend of 12/16 - 12/17
4. ETA: I am anticipating another week to finish or make significant progress through the ticket.

**Comment by roychan1** [comment]
1. Progress: I have provided findings of the medium issue I am working on in a comment, and moved it to "Questions/ in review."
2. Blockers: N/A
3. Availability: day time during the week of 12/31 - 1/6
4. ETA: 1-2 weeks, depending on status of the current medium issue and the large issue I pick up after.

**Comment by roychan1** [comment]
1. Progress: I have picked up a large issue today and will start working on it.
2. Blockers: N/A
3. Availability: I am available 8am-5pm most days
4. ETA: 1 week

**Comment by ExperimentsInHonesty** [comment]
Has been assigned to a Large issue since January 12 (almost 7 weeks).  If no response on his issue by end of next week, unassign and close this prework as inactive.

**Comment by roychan1** [comment]
1. Progress: Working on a large issue
2. Blockers: I have been unable to work on or update my issue due to personal reasons, I apologize about that
3. Availability: 9am-5pm
4. ETA: 1 week

**Comment by roychan1** [comment]
1. Progress: Working on large ticket
2. Blockers: busy schedule
3. Availability: 8am-5pm
4. ETA: 2 weeks

**Comment by roychan1** [comment]
1. Progress: Working on large ticket
2. Blockers: busy schedule
3. Availability: 8am-5pm
4. ETA: 2 weeks

**Comment by ExperimentsInHonesty** [comment]
@roychan1 I don't see you assigned to a large size issue anymore.  

Are you still on the project? If not, please let us know by leaving a comment on this issue and closing it.

If you are still on the project, what is the issue you are working on?

**Comment by HackforLABot** [comment]
@roychan1

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 13, 2024 at 12:14 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roychan1

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 27, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
The Hack for LA Bot has closed this issue due to member inactivity.

**Comment by HackforLABot** [comment]
## Activity Log: roychan1
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- roychan1 opened: https://github.com/hackforla/website/issues/5781 at 10/24/2023, 08:00 PM PDT
- roychan1 assigned: https://github.com/hackforla/website/issues/5781 at 10/24/2023, 08:00 PM PDT
- roychan1 assigned: https://github.com/hackforla/website/issues/5754 at 10/26/2023, 10:30 PM PDT
- roychan1 closed by PR 5793: https://github.com/hackforla/website/issues/5754#event-10787516151 at 10/26/2023, 10:49 PM PDT
- roychan1 commented on: https://github.com/hackforla/website/issues/5754#issuecomment-1782334585 at 10/26/2023, 10:49 PM PDT
- roychan1 reopened: https://github.com/hackforla/website/issues/5754#issuecomment-1782334585 at 10/26/2023, 10:49 PM PDT
- roychan1 commented on: https://github.com/hackforla/website/issues/5754#issuecomment-1782348400 at 10/26/2023, 11:05 PM PDT
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1784029308 at 10/29/2023, 01:09 AM PDT
- roychan1 opened: https://github.com/hackforla/website/pull/5793 at 10/29/2023, 02:55 PM PDT
- roychan1 PR was merged: https://github.com/hackforla/website/pull/5793#event-10816023533 at 10/30/2023, 11:00 PM PDT
- roychan1 issue closed by PR 5793: https://github.com/hackforla/website/issues/5754#event-10816023774 at 10/30/2023, 11:00 PM PDT
- roychan1 assigned: https://github.com/hackforla/website/issues/5792 at 11/03/2023, 12:57 AM PDT
- roychan1 commented on: https://github.com/hackforla/website/issues/5792#issuecomment-1792010981 at 11/03/2023, 01:00 AM PDT
- roychan1 opened: https://github.com/hackforla/website/pull/5837 at 11/05/2023, 01:55 AM PST
- roychan1 PR was merged: https://github.com/hackforla/website/pull/5837#event-10865936602 at 11/05/2023, 01:42 PM PST
- roychan1 issue closed by PR 5837: https://github.com/hackforla/website/issues/5792#event-10865936631 at 11/05/2023, 01:42 PM PST
- roychan1 assigned: https://github.com/hackforla/website/issues/5857 at 11/09/2023, 11:24 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5857#issuecomment-1805227216 at 11/09/2023, 11:26 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1805231209 at 11/09/2023, 11:30 PM PST
- roychan1 opened: https://github.com/hackforla/website/pull/5918 at 11/17/2023, 12:10 AM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1815916493 at 11/17/2023, 12:15 AM PST
- roychan1 PR was merged: https://github.com/hackforla/website/pull/5918#event-11041437759 at 11/22/2023, 01:36 PM PST
- roychan1 issue closed by PR 5918: https://github.com/hackforla/website/issues/5857#event-11041437878 at 11/22/2023, 01:36 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1826241286 at 11/24/2023, 11:35 PM PST
- roychan1 assigned: https://github.com/hackforla/website/issues/5897 at 11/24/2023, 11:43 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5897#issuecomment-1826246011 at 11/24/2023, 11:45 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1842020951 at 12/05/2023, 07:21 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5897#issuecomment-1848370200 at 12/09/2023, 02:55 AM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1857386422 at 12/14/2023, 11:12 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5897#issuecomment-1858763484 at 12/16/2023, 12:42 AM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5897#issuecomment-1873185786 at 12/31/2023, 10:37 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1873186500 at 12/31/2023, 10:40 PM PST
- roychan1 issue closed as completed: https://github.com/hackforla/website/issues/5897#event-11394559228 at 01/04/2024, 12:24 PM PST
- roychan1 assigned: https://github.com/hackforla/website/issues/3906#issuecomment-1876123987 at 01/12/2024, 09:06 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1890313160 at 01/12/2024, 09:08 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1890313648 at 01/12/2024, 09:11 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1928864984 at 02/05/2024, 10:27 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1970269545 at 02/28/2024, 06:09 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1970273951 at 02/28/2024, 06:15 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1970275080 at 02/28/2024, 06:16 PM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-1986803007 at 03/09/2024, 01:13 AM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1986803286 at 03/09/2024, 01:14 AM PST
- roychan1 commented on: https://github.com/hackforla/website/issues/5781#issuecomment-2036333091 at 04/03/2024, 11:55 PM PDT
